pitest-entry/src/main/java/org/pitest/mutationtest/config/ReportOptions.java

```diff
@@ -78,7 +75,7 @@
   private Collection<String>             excludedClasses                = Collections
       .emptyList();
 
-  private Collection<Predicate<String>>  excludedTestClasses            = Collections
+  private Collection<String>             excludedTestClasses            = Collections
```

Comment:
This is quite a large PR. but largely easy to merge as it is just replacing legacy collection manipulation with more modern ones.

A signature/type change is different. I'm not sure if switching to a Collection<String> is desirable, or if it has wider consequences. I'd prefer not to have to think about that before merging this PR, could you revert this?

Reply:
I don't have a lot of experience with Github reviews, but if I'm not missing anything, that hasn't been done yet, has it?

Reply:
No, please see my https://github.com/hcoles/pitest/pull/738#issuecomment-622063578. I'm not going to spend any more time on this PR.